### PR TITLE
Cosmetic changes

### PR DESCRIPTION
**Pull request by Oedipsos**
- Harmonize namings, labels, etc.
- Added space to new commands/environments.
- Added needed arguments for readability

### CODE REVIEW

**Comment by Oedipsos** [comment]
Also added my name in authors and made a command for median point ;)

File: notesdecours.tex [hunk 1]
```diff
@@ -10,88 +12,111 @@
 \usepackage{caption}
 \usepackage{multirow}
 \usepackage{physics}
+\usepackage{braket} % `physics' already implements braket notation, which one should we use?
```
Comment (jmoeil) [inline notesdecours.tex:15]: I’ll try some things tonight to see the difference between the two.

**Review by jmoeil — COMMENTED** [review]
I’ll try to see the difference between `\usepackage{braket} ` and the brakets in `\usepackage{physics}` so we can move forward with the best out of the two.

**Review by jmoeil — CHANGES_REQUESTED** [review]
I tested a bit. There seems to be no graphical difference between `\bra`,`\ket` or `\braket` between the `physics` package or the `braket` package. I therefore propose to only use `physics` to lighten up the `notesdecours.tex` file as well as to stay consistant throughout the code.
I let you have the pleasure of removing line 15 from the code.

**Review by jmoeil — APPROVED** [review]

**Pull request merged**